src/pages/location/createLocation/CreateLocationPage.ts

```diff
@@ -4,17 +4,20 @@ import { expect } from '@/fixtures/fixtures';
 import BasePageModel from '@/pages/BasePageModel';
 import LocationConfigurationTabSection from '@/pages/location/createLocation/tabs/LocationConfigurationTabSection';
 import LocationDetailsTabSection from '@/pages/location/createLocation/tabs/LocationDetailsTabSection';
+import BinLocationsTabSection from './tabs/BinLocationsTabSection';
```

Comment:
Can you change it to contain `@` in the import? We should avoid relative imports